packages/static-wado-deploy/bin/deploydicomweb.mjs

```diff
@@ -1,5 +1,6 @@
-import configureProgram from "../lib/program/index.mjs";
-import { deployConfig } from "../lib/index.mjs";
+#!/usr/bin/env node
+
```

Comment:
This command has a problem with s3-deploy package. When executed, it gives  

Cannot find module '/app/node_modules/@radicalimaging/s3-deploy/build/lib/s3.plugin.mjs' imported from /app/packages/static-wado-deploy/lib/DeployGroup.mjs